proc_nutation.py

```diff
@@ -16,6 +16,7 @@
 logger = init_logging("info")
 max_kHz = 200
 for searchstr, exp_type, nodename, postproc, freq_slice, t_slice in [
+        # PR: either choose to process all, or only one
```

Comment:
deal with this

Reply:
wrong script has since been deleted (was not in examples repo and very old)